backend/battling/views.py

```diff
@@ -39,21 +42,28 @@ class CreateTeam(UpdateView):
     model = Team
     form_class = CreateTeamForm
     template_name = "battling/create_team.html"
-    success_url = reverse_lazy("home")
 
     def form_valid(self, form):
         battle = self.get_object().battle
-        battle_creator = battle.creator
+        form.save()
 
-        if self.request.user == battle_creator:
-            messages.success(self.request, "Your battle was created!")
+        creator = Team.objects.filter(battle=battle, trainer=battle.creator.id)
+        creator_pokemon = PokemonTeam.objects.filter(team=creator[0])
+        creator_team = [pokemon.pokemon for pokemon in creator_pokemon]
```

Comment:
Those 3 lines are used to get the "pokemons of the creator in this battle".

I think this can be done in a simpler way. Like this:
```python
pokemons = list(Team.objects.filter(battle=battle, trainer=battle.creator).prefetch_related('pokemons'))
```

LMK if you don't understand or it doesn't work.

Reply:
Got it! Thanks :D